packages/react-components/react-provider/src/components/FluentProvider/FluentProvider.types.ts

```diff
@@ -23,7 +24,7 @@ export type FluentProviderProps = Omit<ComponentProps<FluentProviderSlots>, 'dir
 export type FluentProviderState = ComponentState<FluentProviderSlots> &
   Pick<FluentProviderProps, 'targetDocument'> &
   Required<Pick<FluentProviderProps, 'dir'>> & {
-    theme: Theme | Partial<Theme> | undefined;
+    theme: ThemeContextValue;
```

Comment:
There were 5 or 6 instances of this specific typing, so I extracted it.

Reply:
Is this related to the useTheme change?